bitcoin/chainparams.c

```diff
@@ -34,6 +34,7 @@ const struct chainparams networks[] = {
 				      0x5a, 0x08, 0x9c, 0x68, 0xd6, 0x19, 0x00,
```

Comment:
I really like to have this type of offset for the network port, and I think that using this technique is pretty good. However, I things in this case we are mixing the bitcoin config with only one propriety related to c-lightning.

However, we can derive your offset from the following formula `lightning_port = defautl_lightning_port + (bitcoin_network_port - bitcoin_mainet_port)`. As an example we can have the following example:

`lightning_port = 9735 + (18332 - 8332)` where we will have that the lightning port is 19735.

What do you think?

Reply:
So do you recommend to do `port_add = 18332 - 8332` instead of `port_add = 10000`?

Reply:
Check now. bbc94acef

Reply:
What's the corner case? I tested it on mainnet and signet. And also with a port `12345` specified on the command line. Works well.

Reply:
And even better, see b525259b0.

Reply:
And now with a comment: 17a66f421

Enough. Will wait for feed-back now.

Reply:
See where one test was fixed along with the `rpc_port` for `regtest` network.